### DISCUSSION TITLE
Please, help! MongoServerError: Caused by :: Write conflict during plan execution and yielding is disabled.

### DISCUSSION THREAD

**Discussion by vstarush** [payloadcms/payload#7053] [Q&A]
Hello,

Please help! I am having this issue popping up occasionally and I can not figure out how to deal with this.
Payload version: 2.11.2

Error: MongoServerError: Caused by :: Write conflict during plan execution and yielding is disabled. :: Please retry your operation or multi-document transaction.

I would appreciate any feedback!
Thank you!

**Comment by DanRibbens** [comment]
I'd be curious if you still run into this, especially if you've upgraded to 3.0 beta.

**Reply by PP-Tom** [reply]
This is happening a lot on beta 3.0 beta, it's happening on Payload Cloud

https://discord.com/channels/967097582721572934/967097582721572937/1300819110007013498
https://discord.com/channels/967097582721572934/967097582721572937/1301615184011333794
https://discord.com/channels/967097582721572934/1215659716538273832/1303105738486317178
https://discord.com/channels/967097582721572934/1215659716538273832/1248023885102710815

**Comment by budescode** [comment]
Hey, did you find any solution?

**Reply by tomtana** [reply]
I had the same issue popping up with version 3.9.0 when processing concurrent stripe webhooks. I could not find out why this issue occurring but you can "fix" it by disabling transactions completely. 

See [here](https://payloadcms.com/docs/database/mongodb#options) in the docs for more Infos.

In your payload config:
```javascript
export default buildConfig({
// all your config
db: mongooseAdapter({
    url: process.env.DATABASE_URI || '',
    transactionOptions: false,
  }),
// rest of config
})
```

**Reply by MinSomai** [reply]
is this good? like what would be the long term impacts

**Comment by cgilly2fast** [comment]
I get this error all the time especially when auto save it on or doing deletes like this: ``` await payload.delete({
        collection: 'file-chunks',
        where: {
          fileId: {
            equals: input.fileId,
          },
        },
        overrideAccess: true,
      })``` It doesn't even through normal like try catches doesnt catch it @DanRibbens
